packages/core/package.json

```diff
@@ -188,23 +188,6 @@
   "bin": {
     "keystone": "bin/cli.js"
   },
-  "files": [
-    "session",
-    "bin",
-    "___internal-do-not-use-will-break-in-patch",
-    "next",
-    "schema",
-    "scripts",
-    "dist",
-    "static",
-    "admin-ui",
-    "testing",
-    "fields",
-    "types",
-    "system",
-    "context",
-    "access"
-  ],
```

Comment:
I am OK with this, we don't stipulate exactly what files are in other packages.
For defining our public exports, we use `"exports"`.

@borisno2 any objections?

Reply:
LGTM @dcousens - the only possible concern would be bundle size, however, given the `src` directory is only 1MB  I think there are better ways to reduce that.

Reply:
@borisno2 I am only seeing 168K for the `src` when compressed